.github/workflows/update-internal-mirrors.yaml

```diff
@@ -2,15 +2,92 @@ name: Update Mirrors
 on:
   push:
     tags:
-      - v*
       - main
   workflow_dispatch:
+  schedule: # add schecule for once a week on saturday
+    - cron: '0 0 * * 6'
 
 concurrency:
   group: update-mirrors-${{ github.ref }}
   cancel-in-progress: true
 jobs:
   update-mirrors:
+    runs-on: ubuntu-latest
+    environment: integration
+    strategy:
+      fail-fast: false
+      matrix:
+        mirror:
+          # note library just means it is part of the official library
+          - name: library/postgres
+            expression: '^[0-9]+\.[0-9]+$'
```

Comment:
so that means it will download all major and minor versions?

Reply:
The script only pulls the latest 50 releases so in reality it only finds the latest release for each lts version:
16.2
15.6
14.11
13.14
12.18
Example from when I was testing the action: https://github.com/smartcontractkit/chainlink-testing-framework/actions/runs/8085942573/job/22094567790#step:5:40